client/test/app/queue/AddCavcRemandView.test.js

```diff
@@ -55,6 +57,24 @@ describe('AddCavcRemandView', () => {
     expect(descisionIssues.map((issue) => issue.id).every((id) => decisionIssueChecks[id])).toBeTruthy();
   });
 
+  describe('"Are judgement and mandate dates provided?" input', () => {
```

Comment:
```suggestion
  describe('Are judgement and mandate dates provided?', () => {
```
Think this is all you need here for this one